wiki/Usage_permissions/en.md

```diff
@@ -0,0 +1,427 @@
+---
+tags:
+  usage
+  permissions
+---
+
+# Usage permissions
+
+## Opt-in
+
+If you are an artist and wish to have your stance regarding whether osu! users may make use of your work known publicly or privately, please feel free to reach out to either [the osu! support team via email](mailto:support@ppy.sh) or any member of the [osu!team](/wiki/People/osu!team) with as much information as possible regarding what work you may want to allow or disallow.
+
+## Audio
+
+All tracks listed on any [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists) are approved for use in the creation of osu! beatmaps. Tracks from a Featured Artist that do not appear in their listing are *not* licensed, and additional permissions may be required to use them.
```

Comment:
this feels more like a *second* sentence or paragraph of this section rather than the first -- something needs to first explain what this section is generally about (the table), and then this note about FA can come in to clarify how things work in that case.